src/main/java/org/sagebionetworks/bridge/reporter/worker/BridgeReporterProcessor.java

```diff
@@ -91,8 +71,13 @@ public void callback(String messageBody) throws IOException, PollSqsWorkerBadReq
                 ObjectNode reportData = JsonNodeFactory.instance.objectNode();
                 String studyId = studySummary.getIdentifier();
 
-                // get all uploads for this studyid
-                ResourceList<Upload> uploadsForStudy = bridgeHelper.getUploadsForStudy(studyId, startDateTime, endDateTime);
+                // get all uploads for this studyid, differentiating by scheduleType
+                ResourceList<Upload> uploadsForStudy = null;
+                try {
+                    uploadsForStudy = getUploadsForStudyHelper(studyId, startDateTime, endDateTime, scheduleType);
+                } catch (InterruptedException e) {
+                    LOG.error("InterruptedException: " + e);
+                }
```

Comment:
If this throws, the next line will immediately throw a NullPointerException. Would it be better to throw an exception here so the NullPointerException is less mysterious?

Also, I assume this InterruptedException is from Thread.sleep(). If so, would it be better to just have the try-catch around the sleep() and just log the error message and move on?

Lastly, is it worth preserving the stack trace? You need LOG.error(message, e)